data_process.py

```diff
@@ -339,6 +339,11 @@ def add_x_t_data(files, path, x_length, t_length, start_date,
     return train, val, test
 
 
+# %% Normalize data functions
+
+# TODO ADD HERE
```

Comment:
Can add the new functions here in this cell and above "# %% Augment data functions"